src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,39 @@ public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
+
+  public static class DriveTrainConstants {
+    public static final int frontLeftMotorID = 7;
+    public static final int backLeftMotorID = 6;
+    public static final int frontRightMotorID = 8;
+    public static final int backRightMotorID = 5;
+    public static final int kDriverControllerPort = 1;
+    public static final int JoystickID = 0;
+  }
+
+  public static class ShooterConstants {
+    public static final int frontShooterId = 7;
+    public static final int backShooterId = 8;
+
+    public static final double frontShooterSpeed = .5;
+    public static final double frontShooterSpeedReverse = -.5;
+
+    public static final double backIndexSpeed = .5;
+    public static final double backIndexSpeedReverse = -.5;
+
+    public static final double innerSpeed = .5;
+    public static final double outerSpeed = .5;
+
+    public static final double innerSpeedReversed = -.3;
+    public static final double outerSpeedReversed = -.3;
```

Comment:
@CarloWoolsey would it make sense to just have 2 variables instead of the 4, since innerSpeed and outerSpeed are static and the same value, same for innerSpeedReversed and outerSpeedReversed?
```suggestion
    public static final double innerOuterSpeed = .5;
    public static final double innerOuterSpeedReversed = -.3;
```

Reply:
Potentially --please reference my initial comment re-outer speed & inner speed fields. If the fields will always be set to the same value, I don't see the need for multiple fields (for a single command). With this said, we will most likely need to tune different roller speeds for different shooter positions. 
To clear things up further, though,  I would change variable names to reference the specific command they will be used for (e.g. `intakeSpeed`, etc.). 
Please let me know if this makes sense.

Reply:
@CarloWoolsey I think your comment above makes sense. However I am unsure of what the speeds are for? are they for the front and rear rollers? is `outerspeed` supposed to refer to the front rollers and `innerSpeed` for the rear rollers?
If so, I was under the impression that the rear rollers and front rollers were supposed to turn at different speeds for some reason.

Reply:
We're on the same page --I would move forward with the changes requested in your initial comment.